infrastructure.tf

```diff
@@ -257,6 +257,21 @@ resource "oci_identity_tag" "openshift_instance_role" {
   provider = oci.home
 }
 
+resource "oci_identity_tag" "openshift_resource" {
+  description      = "Openshift Resource"
+  is_cost_tracking = "false"
```

Comment:
The field enables customers to track costs using this tag. So if we enable this, the customers could not only use this tag for managing the infra but also for tracking usage based on the tag. 

Please make this **true**

Reply:
makes sense. done